RELEASING.md

```diff
@@ -90,19 +90,14 @@ Create a pull request titled `chore: x.y.z release proposal`. The commit body sh
 
 Merge the PR, and pull the changes locally (using the commands in the first step). Ensure that `chore: x.y.z release proposal` is the most recent commit.
 
-## Publish all packages
+## Compile all packages
```

Comment:
note to reviewer: IMO this step was missing since `npm publish` will pack and publish the packages in whatever state it is. With a full compilation everything should be in the right state for publishing